src/app/landing/landing.component.scss

```diff
@@ -0,0 +1,126 @@
+.landing_page_info {
+  opacity: 1;
+  z-index: 2;
+  padding-left: 84px;
+  padding-top: 250px;
+  width: 60%;
+  display: flex;
+  flex-direction: column;
+  color: var(--main-white);
+  .landing_page_title {
```

Comment:
Since the `*.scss` is scoped to the component - you don't need to nest these classes. You can define them on the same level:
```scss
.landing_page_info {
//...
}
.landing_page_title {
//...
}
//etc..
```